### ISSUE TITLE
Support for FK33

### ISSUE THREAD

**Issue by qinapps** [RainbowMiner/RainbowMiner#1595]
Is it possible do add support for the FK33? It would be so lovely. I can help and support I have already FK33 be me

**Comment by RainbowMiner** [comment]
Do you have any documentation regarding that Forest Kitten 33 boards and/or API resp. controls?

**Comment by qinapps** [comment]
yes its lucky very simple, its based on ethminer the only extra think on windows is that the fdti driver must be installed and a brigde application musst running

**Comment by RainbowMiner** [comment]
> yes its lucky very simple, its based on ethminer the only extra think on windows is that the fdti driver must be installed and a brigde application musst running

So, you simply start a special ethminer clone? Do you have a download link and specifications? If yes - please tell me.

**Comment by qinapps** [comment]
The software can found here: https://github.com/SquirrelsResearch/eth-release-pkg
I tested in the dir eth-release-pkg/1.7_Beta2/
so on windows i running start_bridge.bat and start_eth.bat (on linux the .sh files)
The brigde app check is the fpga with the right bitstream loadded, when not it will automatical load the bitstream

**Comment by qinapps** [comment]
I have here fk33 so I can give you access to this pc when you want (I send you a PM on Discord)

I belive after the press release from yesterday from NVIDIA is FPGA the new way to go :)

**Comment by RainbowMiner** [comment]
Nice. That helps.

**Comment by f8vze** [comment]
@qinapps what sort of performance are you seeing with the fk33?

**Comment by RainbowMiner** [comment]
RainbowMiner orchestrates CPU/GPU miner binaries and has no FPGA support or a path towards it, and the linked FK33 project is gone as well (404). Closing as not planned - thanks for the suggestion!
